.github/workflows/kuttl_tests.yaml

```diff
@@ -72,53 +72,56 @@ jobs:
       matrix:
         k8s_version: ["1.21"]
         kuttl-test:
-        - test-servicemonitors
-        - test-user-defined-ns
-        - test-all-deployments
+          - test-servicemonitors
+          - test-user-defined-ns
+          - test-all-deployments
     steps:
-    - name: Check out code into the Go module directory
-      uses: actions/checkout@v2
-      if: github.event_name == 'pull_request'
-      with:
-        ref: ${{ github.event.pull_request.head.sha }}
-    - uses: actions/checkout@v2
-      if: github.event_name != 'pull_request'
-    - name: Set up Go
-      uses: actions/setup-go@v3
-      with:
-        go-version-file: 'go.mod'
-        cache: true
-    - name: Kind kube-proxy issue workaround
-      run: sudo sysctl net/netfilter/nf_conntrack_max=524288
-    - name: Download k8ssandra-operator image
-      uses: actions/download-artifact@v2
-      with:
-        name: k8ssandra-operator
-        path: /tmp
-    - name: Load Docker images
-      run: |
-        docker load --input /tmp/k8ssandra-k8ssandra-operator.tar
-    - name: Install kustomize
-      uses: imranismail/setup-kustomize@v2
-    - name: Install kuttl
-      run: |
-        make install-kuttl
-    - name: Create kind cluster
-      run: |
-        kind create cluster --name k8ssandra-0 --config ./test/kuttl/config/kind/w3k${{ matrix.k8s_version }}.yaml
-        make cert-manager
-        make IMG="k8ssandra/k8ssandra-operator:${{ needs.build_image.outputs.image_tag }}" kind-load-image
-    - name: Run kuttl test
-      run: |
-        # We are running tests against k8s 1.21
-        # Additional versions must be added in kind config files under ./test/config/kind
+      - name: Make sure kustomize is installed in the place the kuttl tests expect
+        run: |
+          make kustomize
```

Comment:
I think someone must have changed the way that kustomize is installed and the kuttl tests are now failing on every PR. This change fixes that problem.

We've previously discussed that we should;

a) Have the same CI workflow locally and on GHA.
b) Not install things globally for local machines. 

Those constraints mean that we need to do things this way. Also, I don't understand why we'd be saying that kustomize is a hard dependancy but a GNU grep isn't :p

Reply:
NB: we could also try modifying the path in the makefile so that kustomize always runs from the local `bin` directory. That would avoid us needing to use so many ugly relative paths if that's the concern. But installing kustomize via Github actions will cause inconsistencies between the local and remote CI processes.

Reply:
I removed the changes RE kustomize, we'll address these problems as part of a separate PR.